packages/config/config/devices/0x0129/ntm625.json

```diff
@@ -0,0 +1,358 @@
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 255
+				}
+			]
+		},
+		"15": {
+			"label": "Reset To Factory Defaults",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"16": {
+			"label": "Escape Return Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 255
+				}
+			]
+		},
+		"18": {
+			"label": "Door Propped Timer",
+			"description": "10 to 2540 second. This value is represented as seconds X 10. (ie a value of 4 would mean a door propped timer of 40 seconds)",
+			"unit": "decaseconds",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 254,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"19": {
+			"label": "DPS Alarm (Ax sensor)",
+			"description": "Used to enable/disable door condition alarms for locks that support a door position sensor. Any lock that does not support DPS will always report 0x00.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 255
+				}
+			]
+		},
+		"20": {
+			"label": "Deadbolt Installed",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": true,
+			"writeOnly": false,
+			"allowManualEntry": false
+		},
+		"21": {
+			"label": "Eco Mode",
+			"description": "If supported switches lock to lower power mode of operation with reduced sound and led brightness.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 255
+				}
+			]
+		},
+		"22": {
+			"label": "Privacy Mode With Deadbolt",
+			"description": "If enabled then extending the deadbolt will put the lock into Privacy Mode.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"23": {
+			"label": "Lock Body Alarms Mask",
+			"description": "Each bit controls one alarm. 1 = alarm enabled, 0 = alarm disabled. 0xFF = All Alarms on, Default is 0xFF. Bit 0: DPS Alarm (AX) (Duplicate of parameter 19. Values should be tied together) Bit 1: Deadbolt Alarm Bit 2: Lever Rotated (Rx) Bit 3 – 7: Not Used",
```

Comment:
Judging from the description, this should be modeled as [partial parameters](https://zwave-js.github.io/node-zwave-js/#/development/config-files?id=partial-parameters).

Reply:
Note that bit 2 does not mean value = 2

Reply:
I'll make an attempt.